client/src/redux/prop-types.ts

```diff
@@ -129,6 +129,20 @@ export interface VideoLocaleIds {
   portuguese?: string;
 }
 
+export type ChallengeWithCompletedNode = {
+  block: string;
+  challengeType: number;
+  dashedName: string;
+  fields: {
+    slug: string;
+  };
+  id: string;
+  isCompleted: boolean;
+  order: number;
+  superBlock: string;
```

Comment:
Although, can we change this to:

```suggestion
  superBlock: SuperBlocks
```